### DISCUSSION TITLE
Electric Blocks

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#149] [Showcase]
![elctric-blocks](https://user-images.githubusercontent.com/3902872/108853006-83da9400-75e6-11eb-9c24-8905139e2efc.png)

> Electric Blocks is a Minecraft mod that revolutionizes power engineering.
> 
> More specifically, Electric Blocks is a Minecraft Forge mod that allows you to run realistic power flow simulations for educational and engineering purposes. This site contains documentation for using this mod.
> 
> This mod is sponsored by the University of Idaho as part of the Capstone Design project.

👉 [electricblocks.github.io](https://electricblocks.github.io/)

**Comment by h-enk** [comment]
Hey @zachoooo, nice-nice! You OK with me adding the Electric Blocks website to the [Doks showcase](https://getdoks.org/showcase/)?

**Reply by zachoooo** [reply]
Absolutely! Thanks you very much for your work on this project.

**Reply by h-enk** [reply]
There it is! https://getdoks.org/showcase/electric-blocks/
